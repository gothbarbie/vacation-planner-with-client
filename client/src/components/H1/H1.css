.h1 {
  font-size: 32px;
  color: #333333;
  padding-bottom: 1rem;
  margin-top: 3rem;
  height: 4rem;
}

.h1 span {
  padding-bottom: 1rem;
  border-bottom: 3px solid #80D1FF;
}

.h1--light {
  color: #f2f2f2;
}

.h2--dark {
  color: #333333;
}
