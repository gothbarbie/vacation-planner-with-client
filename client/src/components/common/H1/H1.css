.h1 {
  font-size: 32px;
  color: #333333;
  padding-bottom: 1rem;
  border-bottom: 3px solid #80D1FF;
  margin-top: 1rem;
}

.h1--light {
  color: #f2f2f2;
}

.h2--dark {
  color: #333333;
}
