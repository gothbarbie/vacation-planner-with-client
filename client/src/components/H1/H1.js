// @flow

import * as React from 'react'
import classnames from 'classnames'

import './H1.css'

type props = {
  children: React.ChildrenArray<any>,
  light?: boolean,
  dark?: boolean,
}

const H1 = ({ children, light, dark }: props) => (
  <h1
    className={classnames('h1', {
      'h1--light': light,
      'h1--dark': dark,
    })}>
    <span>{children}</span>
  </h1>
)

export default H1
