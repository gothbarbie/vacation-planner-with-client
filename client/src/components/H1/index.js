// @flow

import * as React from 'react'
import classnames from 'classnames'

import './H1.css'

type props = {
  children: React.ChildrenArray<any>,
  light?: boolean,
  dark?: boolean
}

export default ({ children, light, dark } : props) => (
  <h1 className={classnames('h1', {
    'h1--light': light,
    'h1--dark': dark,
  })}><span>{children}</span></h1>
)
