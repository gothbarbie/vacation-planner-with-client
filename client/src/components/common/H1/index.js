// @flow

import React from 'react'
import classnames from 'classnames'
import type { Element } from 'react'

import './H1.css'

type props = {
  children: Element<any>,
  light?: boolean,
  dark?: boolean
}

export default ({ children, light, dark } : props) => (
  <h1 className={classnames('h1', {
    'h1--light': light,
    'h1--dark': dark,
  })}><span>{children}</span></h1>
)
