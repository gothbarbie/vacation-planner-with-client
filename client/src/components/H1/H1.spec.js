import React from 'react'
import { shallow } from 'enzyme'

import H1 from './H1'

describe('components/H1', () => {
  const h1 = shallow(<H1 />)

  it('renders', () => {
    expect(h1).toMatchSnapshot()
  })
})
