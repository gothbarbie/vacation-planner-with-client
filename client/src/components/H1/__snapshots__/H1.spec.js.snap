// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/H1 renders 1`] = `
<h1
  className="h1"
>
  <span />
</h1>
`;
